src/i18n.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @description An array of locales
+ * language (ISO 639-1 - set 1): https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes
+ * country code (ISO 3166-1 alpha-2): https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+ * E.g.: Canadian French would be: `fr-ca`
+ */
+export const SUPPORTED_LOCALES = ["pt-br"];
+
+export function missingTranslationMessage(path: string) {
+	switch (path) {
+		case "pt-br":
```

Comment:
Nitpick: including "pt-pt" here assumes we have another supported locale but that one is not on the `SUPPORTED_LOCALES` array. Would it make sense somehow to tie this array and this function together?

Reply:
one is about having language translations, the other is about having languages as supported.

I consider neither of those languages supported. But I had to add a locale there so TypeScript would give me a break...I didn't want to add a Portuguese entry on _this_ PR, though - I can add on a follow-up, at least for an entry